container-scan/README.md

```diff
@@ -0,0 +1,58 @@
+# Lacework Container Scan
```

Comment:
We are using action-docs to automatically generate docs for actions. If you want to try to setup that here https://github.com/open-turo/actions-jvm/blob/993178f87d51d3c8f972403d58deb8ae59980fff/.pre-commit-config.yaml#L24